httplib.h

```diff
@@ -2660,25 +2658,27 @@ inline bool bind_ip_address(socket_t sock, const char *host) {
 
 #ifdef USE_IF2IP
 inline std::string if2ip(int address_family, const std::string &ifn) {
-  struct ifaddrs *ifap;
+  struct ifaddrs *ifap = nullptr;
```

Comment:
`ifap` is initialized with `getifaddrs(&ifap)` below, setting it to `nullptr` is not needed.